src/database/utils/migrationLoader.ts

```
@@ -0,0 +1,105 @@
+
+export class MigrationLoader {
+  /**
+   * Load a migration SQL file from the file system
+   * @param filePath - The path to the migration file
+   * @returns The SQL content of the migration file
+   */
+  static async loadMigrationFile(filePath: string): Promise<string> {
+    try {
+      // In a real implementation, this would use Node.js fs module
+      // For now, we'll return the SQL content based on the file path
+      return this.getMigrationSQL(filePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)